src/js/utils/background.js

```diff
@@ -43,19 +43,20 @@ export const backgroundIsDark = (backgroundArg, theme) => {
   return result;
 };
 
-export const backgroundStyle = (backgroundArg, theme) => {
+export const backgroundStyle = (backgroundArg, theme, componentTextColor) => {
```

Comment:
I'd probably just call this `textColor`. I don't think the `component` prefix is needed, just like it isn't `componentBackgroundArg`.

Reply:
maybe textArg ? to align better. textColor is already being used.

Reply:
I'd choose the argument names first and then decide on internal variable names.